### ISSUE TITLE
Description on every forum page

### ISSUE THREAD

**Issue by cptX** [backdrop-contrib/forum#32]
For helping the users, giving them information or instructions, ensuring they are in the correct forum, but also for enhancing the SEO capability of every forum page it would be very nice to have the forum description under the title and above the "Add new Forum topic" link possibly as a h2/h3 header.

How can I achieve this?
I could build a new block programmatically for example, but is there any other way without messing with the code?
I read about drupal 7 that by using tokens module you could use tokens inside text areas? Could this be used to print the description inside a new block?

Could you consider it as feature request for future releases?

**Comment by indigoxela** [comment]
Hi @cptX,
welcome to the Backdrop community!

I'm not sure if I understand your request correctly, but it seems that the [Token Filter](https://backdropcms.org/project/token_filter) modules could do what you want. (Tokens are in core, but the filter to replace tokens in content ships with that module.)

Again, I'm not sure, but it's worth a try, I guess.

**Comment by cptX** [comment]
Well first of all this is the first time I'm writing to the backdrop community and I'm impressed from your response time. In Drupal could take months to get a response, so thanks for that.
I'll try the token module. Any other possibilities without installing anything extra?
For example there is a block called "existing content" but I don't know what to put at the "Content title or ID" to reference the forum description.

**Comment by indigoxela** [comment]
> I'm impressed from your response time.

That's by accident. Response within minutes isn't guaranteed :smile:

> Any other possibilities without installing anything extra?

I'd almost bet there is, but I'm almost sure, I misunderstand something.

> there is a block called "existing content" but I don't know what to put at the "Content title or ID"

Ah, on a layout form, right? That references existing nodes (hence also Forum topics), the ID is the node ID (=nid) For a node with internal path "node/3" the ID is "3", the title autocompletes the node's title. Should be quite comfortable, actually.

**Comment by cptX** [comment]
Yes but how can I reference the taxonomy description? Does this work only for node ids?

**Comment by indigoxela** [comment]
And as you stated, you're new to Backdrop: maybe [our forum](https://forum.backdropcms.org/forum) is interesting for you.

For both, sneak peak a forum setup, and find help with newbie questions.

**Comment by indigoxela** [comment]
> how can I reference the taxonomy description?

Personally, I'd do it with a views block - but I do everything (too much) with views. I bet, you'll get better hints in our forum. :smiley:

**Comment by cptX** [comment]
I'm not sure if views are able to show the custom icons which change on the status of every topic.

**Comment by cptX** [comment]
Tried with token filter. I'm not able to print anything regarding taxonomy / forum description. Where can I see a list of the available tokens the token filter has?

**Comment by indigoxela** [comment]
Yet another time I'd like to recommend our forum. More people will see your question there. :wink:

**Comment by cptX** [comment]
OK thanks!
